### ISSUE TITLE
[Important] About the Ongoing Maintanance of this Add-on

### ISSUE THREAD

**Issue by yuki-tsubaki** [giovannihenriksen/Anki-Simulator#70]
The last activity on this repository by the maintainer, @giovannihenriksen, (and in fact, their last recorded activity for their entire account) was in the middle of January 2022.

Community activity has not stopped as there were 3 issues and 3 pull requests opened since that time. Even before the last activity, there were multiple unaddressed issues.

Assuming @giovannihenriksen doesn't return, we need to find a new maintainer. I don't have any previous experience publishing Anki add-ons (although, I have hacked/fixed them many times) and am not particularly experienced in UI applications in Python. If anyone more qualified than me is interested in maintaining a fork that would be ideal. That said, if nobody else comes along, than I will at least try to get a proper fork going.

**Comment by Speculate7348** [comment]
I agree. I also have new feature suggestions that could be incorporated by a new maintainer.

**Comment by Yell0whead** [comment]
Hi @yuki-tsubaki and @Speculate7348. It would be amazing if someone does decide to adopt a fork of this add-on to keep it up to date. I would simply like to voice my support!
